sympy/assumptions/predicates/sets.py

```diff
@@ -378,8 +378,25 @@ class TranscendentalPredicate(Predicate):
     transcendental numbers. A transcendental number is a real
     or complex number that is not algebraic.
 
+    Examples
+    ========
+
+    >>> from sympy import ask, Q, pi, exp
+    >>> ask(Q.transcendental(2.5))
+    False
```

Comment:
The doctests are failing because of this.

Reply:
You can run the following from the root directory to recreate the failing test on your computer:
```
bin/doctest --force-colors
```

Reply:
I think it's failing because if this change: [https://github.com/sympy/sympy/pull/25270/commits/48af8e746123a432b72af0af2c2515fa8ead6209](url)
So `ask(Q.transcendental(2.5))` now returns `None` as oppose to `False` earlier. Should I revert back the change?

Reply:
Link to that change doesn't work for me.

I think `ask(Q.transcendental(2.5))` should return None since the old assumptions return None for floats. The docstring example should reflect that.

Also, maybe you can open a new pull request when you're done with this one so that `ask(Q.algebraic(2.5))` returns None.

Reply:
Ok got it.